docs/before/accounts.md

```diff
@@ -18,31 +17,25 @@ To this end, you will need to follow the steps below (*and for more detailed inf
 4. The experiment spokesperson adds your UNIX account to the [Experiment Manager](https://pswww.slac.stanford.edu/lgbk/lgbk/experiments).
 
 !!! note "Enabling access to the S3DF"
-    The SLAC-wide Shared Science Data Facility ([S3DF](https://lcls-users.readthedocs.io/en/latest/glossary/#s3df)) will soon replace the computer and storage resources managed by LCLS Photon Controls and Data Systems ([PCDS](https://lcls-users.readthedocs.io/en/latest/glossary/#pcds)) currently used for LCLS experiment data management.
+    The SLAC-wide Shared Science Data Facility ([S3DF](https://lcls-users.readthedocs.io/en/latest/glossary/#s3df)) has replaced the computer and storage resources managed by LCLS Photon Controls and Data Systems ([PCDS](https://lcls-users.readthedocs.io/en/latest/glossary/#pcds)) that was  used for LCLS experiment data management prior 2023.
     In order to use S3DF, it is necessary to enable your S3DF account using your SLAC UNIX account. Detailed information in the [S3DF documentation here][7].
 
 # Step 2 - Understand LCLS Data Systems
 Now that you can access the computer systems at SLAC, it is time to get yourself oriented. 
 For that, it is useful to understand how the data flows from the beamline to storage, *a.k.a* to understand the LCLS Data Systems, built with 3 successive levels:
 
-| Level # | Processing Time | Storage Duration | Access |
-| -- | -- | -- | -- |
-| 1. Data Acquisition<br> and Reduction | diagnostics within <1s. | None.<br>Data is in shared memory.| **Experts only** |
-| 2. Fast Feedback | diagnostics and results<br> within <1min.| <1 week. | *(before) -* through `pslogin` to `psffb`.<br> *(now) -* on S3DF. |
-| 3. Offline Analysis | intensive compute during<br> and after the beamtime.| <4 months<br> before being moved to tape.| *(before) -* through `pslogin` to `psana`.<br> *(now) -* on S3DF |
+| Level # | Processing Time | Storage Duration | Access                                                                                                                |
+| -- | -- | -- |-----------------------------------------------------------------------------------------------------------------------|
+| 1. Data Acquisition<br> and Reduction | diagnostics within <1s. | None.<br>Data is in shared memory.| **Experts only**                                                                                                      |
+| 2. Fast Feedback | diagnostics and results<br> within <1min.| <1 week. | through `s3dflogin`, then on either interactive or compute node, do `export SIT_PSDM_DATA=/sdf/data/lcls/drpsrcf/ffb` |
+| 3. Offline Analysis | intensive compute during<br> and after the beamtime.| <4 months<br> before being moved to tape.| through `s3dflogin`, then on either interactive or compute node, do `export SIT_PSDM_DATA=/sdf/data/lcls/ds`          |
 
 The figure below provides a high-level map of the LCLS Data Systems presenting the main entities and their networks. (more TBD).
```

Comment:
Do we want to take the time now to detail the difference between LCLS1/LCLS2 DAQ here? The first half of the figure is more applicable to LCLS2 no?

Reply:
That is a good idea!